src/complexity.jl

```diff
@@ -0,0 +1,36 @@
+export ComplexityMeasure
+export complexity
+export complexity_normalized
+
+"""
+    ComplexityMeasure
+
+Abstract type for (entropy-like) complexity measures.
```

Comment:
```suggestion
Supertype for (entropy-like) complexity measures.
```
(that's how its typically called, see e.g. `AbstractDict`)